Game_Manual/Echo.md

```diff
@@ -18,4 +18,47 @@ For many commands, passing the `-h` or `--help` flag will show you a helpful mes
+
+## Guidelines
+
+These guidelines are to support us in making the most of this tool while minimizing distractions and maximizing learning:
+
+Managing **OUTPUT**: guidance for managing the volume of posts & notifications we generate:
+
+1. **Be discerning about WHO you tag - almost never use an @all notification.** Only use an @all notification if it's imperative that everyone in the channel read the message you have sent - assume they will, and that therefore you are interrupting the attention and focus of every person in that channel. @all is the equivalent of ringing the gong. Would you ring the gong for this message? Differentiate between "I'dlikeeveryone to read this" and "Everyonemustread this". Posts in the \#watercooler channel, for example,neverwarrant an @all notification, as it is an entirely recreational / optional channel.
+2. **Be disciplined about WHEN you post.** Do people need to read your message RIGHT NOW? If not, it doesn't warrant a mid-day @all notification. Unless it is urgent or an emergency, do not use an @all notification between the hours of 9:00am and 5:30pm.
+
+3. **Be thoughtful about WHERE you post.** Post information only in the most highly relevant places. If it's about the office space, use the \#space channel. If it's only relevant to November folks, use the cohort channel. If it's about a movie outing you want to plan, use the \#watercooler channel. If you find yourself posting something in three different channels, ask someone's opinion about where it belongs and only post it there. The \#significant-updates channel should be reserved for content that everyone in the Guild must be aware of in a timely fashion. 
+
+4. **Be stingy about creating new channels:** A new channel should only be created for an entirely new topic that will generate ongoing discussion. The more channel clutter there is, the less likely we are to pay attention.
+
+Managing **INPUT**: guidance for managing the volume of posts and notifications you are fielding and protecting your attention throughout the day:
+
+1. **Choose your channels wisely. **Only join channels that you wish to stay up to date with. To ensure smooth Guild-wide communication, please commit to joining and monitoring the following channels, at a minimum: \#Oakland, \#YouHouseChannel, \#YourCohortChannel, \#significant-updates, and your project team channel for the current week. Choose your other channels carefully, hide or leave channels you're done with. Keep it clean. 
+
+2. **Make Echo a "pull" not "push" experience**.** **Decide whether you'd like to turn desktop notifications for Echo off during the day. Schedule set times that you'll catch up on messages - e.g. before standup, over lunch, and right before you leave. Make Echo work for you, don't work for Echo.
```

Comment:
Here it is with the extra little stars removed and an 'a' added:
`**Make Echo a "pull" not a "push" experience**. Decide whether`